src/adapters/openx.js

```diff
@@ -46,6 +46,7 @@ var OpenxAdapter = function OpenxAdapter(options) {
   }
 
   function _requestBids() {
+    if (typeof OX === 'undefined') return;
```

Comment:
I think this breaks the adapter. The first time through `_requestBids` the global `OX` will be undefined because the openx library hasn't been loaded.